### PR TITLE
[skip vbump] prepare for CRAN release

### PR DESCRIPTION
**Pull request by clarkliming**

### CODE REVIEW

**Review by BFalquet — APPROVED** [review]
Thank you, ready for submission

**Comment by BFalquet** [comment]
I think we need to remove the "Remotes" field again.

File: DESCRIPTION [hunk 1]
```diff
@@ -40,7 +40,7 @@ Imports:
     rlistings (>= 0.2.6),
     rtables (>= 0.6.5),
```
Comment (shajoezhu) [inline DESCRIPTION:41]:
```suggestion
    rtables (>= 0.6.6),
```

**Comment by shajoezhu** [comment]
> I think we need to remove the "Remotes" field again.

since everything is now open-source, do you need "remotes" field ever again?

**Comment by clarkliming** [comment]
> > I think we need to remove the "Remotes" field again.
> 
> since everything is now open-source, do you need "remotes" field ever again?

yes I think we can remove it forever

File: DESCRIPTION [hunk 2]
```diff
@@ -14,8 +14,11 @@ Authors@R: c(
     person("Tian", "Fang", , "tian.fang@roche.com", role = "ctb"),
     person("F. Hoffmann-La Roche AG", role = c("cph", "fnd"))
   )
-Description: Standard Table, Listings, and Graphs (TLG) library used in
-    clinical trials.
+Description: Provide standard table, listings, and graphs (TLG) library used in
+    clinical trials. This package implements a stucture to reformat the data with
+    'dunlin', create reporting tables using 'rtables' and 'tern' with standardized
+    input arguments to enable quick generation of standard outputs.
+    In addtion, it also provides comprehensive data checks, script generation functionality.
```
Comment (BFalquet) [inline DESCRIPTION]:
```suggestion
    In addtion, it also provides comprehensive data checks and script generation functionality.
```

**Review by BFalquet — COMMENTED** [review]
minor comment

**Comment by clarkliming** [comment]
@BFalquet I reduce the size of our test data, leading to all the snapshot changes. (it can be further reduced, if needed)

However this should not be critical (as no functionality is updated)

**Review by BFalquet — APPROVED** [review]
The tests for `lbt14` and `chevron-methods` were still a bit slow (~30sec), I trimmed the data in the initial part of the test to speed up the execution to ~15sec locally. `rspt01` snapshot tests were failing but it could be a rounding error, I commit this change separately. Note, tests were performed with the CRAN version of `formatters`, `rlistings`, `rtables`, `tern` and `dunlin`. 

tern_0.9.3, rlistings_0.2.7, rtables_0.6.6, formatters_0.5.5

**Pull request merged**